extensions/amp-story/1.0/amp-story-background-blur.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {setStyles} from '#core/dom/style';
+
+/** @const {number} */
+const CANVAS_SIZE = 3;
+
+/** @const {number} */
+const DURATION_MS = 200;
+
+export class AmpStoryBackgroundBlur {
+  /**
+   * @param {!Window} win
+   * @param {!Element} element
+   */
+  constructor(win, element) {
+    /** @private @const {!Element} */
+    this.element_ = element;
+
+    /** @private @const {!Window} */
+    this.win_ = win;
+
+    /** @private @const {!Element} */
+    this.canvas_ = null;
+
+    /** @private @const {!Element} */
+    this.context_ = null;
+  }
+
+  /**
+   * Setup canvas, context and attach it to the document.
+   */
+  attach() {
+    this.canvas_ = document.createElement('canvas');
+    this.context_ = this.canvas_.getContext('2d');
+    this.canvas_.width = this.canvas_.height = CANVAS_SIZE;
+    setStyles(this.canvas_, {
+      width: '100%',
+      height: '100%',
+      position: 'absolute',
+      left: 0,
+      top: 0,
+    });
```

Comment:
This will either be attached to an `amp-story` or `amp-story-player`. 
I wonder if / how we make a stylesheet that can be visible from either context.

Reply:
Both the `amp-story` and the `amp-story-player` have their own spreadsheet. If you need to share some CSS, you can create a new file and `@import` it in both CSS files. If these 5 properties are all you need, it might be fine to keep in the JS tho.

Reply:
I think this is readable since it's 1 element and 5 properties. If we need to style another element or create more properties we could separate it but this seems really nice and simple.